src/gateway/understanding-kong/how-to/router-atc.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Configure Routes with Expressions
+content-type: how-to
+---
+
+
+With the release of version 3.0, {{site.base_gateway}} now ships with a new router. The new router can describe routes using a domain-specific language called Expressions. Expressions can describe routes or paths as patterns using regular expressions. This how-to guide will walk through switching to the new router, and configuring routes with the new expressive domain specific language, expressions. For a list of all available operators and configurable fields please review the [reference documentation](/gateway/latest/reference/router-operators).
+
+## Prerequisite
+
+* Edit [kong.conf](/gateway/latest/kong-production/kong-conf) to contain the line `router_flavor = atc` and restart {{site.base_gateway}}.
+
+## Create routes with Expressions
+
+To create a new router object using expressions send a `POST` request to the [services endpoint](/gateway/latest/admin-api/#update-route) like this: 
+```sh
+curl --request POST \
+  --url http://localhost:8001/services/example-service/routes \
+  --form  atc='http.path == "/mock"
+```
+
+In this example you associated a new route object with the path `/mock` to the existing service `example-service`. The expressions DSL allows you to create complex router objects using operators and fields. 
+
+```sh
+curl --request POST \
+  --url http://localhost:8001/services/example-service/routes \
+  --header 'Content-Type: multipart/form-data' \
+  --form 'atc=(http.path == "/mock" || net.protocol == "https")'
+```
+In this example the || operator created an expression that set variables for the following fields: 
+
+```
+"protocols": ["http", "https"]
+AND
+"paths": ["/mock"]
+```
+
+You can use non-expressions related attributes within the same `POST` request:
+
+```sh
+curl --request POST \
+  --url http://localhost:8001/services/example-service/routes \
+  --header 'Content-Type: multipart/form-data' \
+  --form 'atc=(http.path == "/mock" || net.protocol == "https") \
+  --form name=mocking
+```
+
+This would define a router object with the following attributes:
+
+
+```
+"atc": "(http.path == \"\/mock\" || net.protocol == \"https\"),
+"name": "mocking",
+
+```
+The list of available attributes is available in the [reference documentation](/gateway/latest/admin-api/#request-body). 
```

Comment:
```suggestion
You can view the list of available attributes in the [reference documentation](/gateway/latest/admin-api/#request-body). 
```
Just rewrote it to avoid multiple repetitions of "available"